pkg/server/server.go

```diff
@@ -4179,11 +4179,11 @@ func (s *BgpServer) WatchEvent(ctx context.Context, r *api.WatchEventRequest, fn
 							l = append(l, p...)
 						}
 						for _, p := range l {
-							pl = append(pl, toPathApi(p, nil, true, false, false))
+							pl = append(pl, toPathApi(p, nil, false, false, false))
```

Comment:
should the same change be applied on the line 4164 (`watchEventUpdate` case), or is there any reason for keeping it there?